wpf/WPF-WinRT/README.md

```diff
@@ -0,0 +1,14 @@
+# WPF-WinRT
+
+This sample shows how to consume the Windows 10 API from .NET Core and .Net Framework projects.
+
+Windows 10 SDK is required.
+
+```xml
+  <ItemGroup>
+    <Reference Include="Windows">
+      <HintPath>$(MSBuildProgramFiles32)\Windows Kits\10\UnionMetadata\10.0.16299.0\Windows.winmd</HintPath>
+      <Private>False</Private>
+    </Reference>
+  </ItemGroup>
+```
```

Comment:
Can you add context to this readme (make it more human readable)?

e.g. To add support for WinRT APIs, add the following to the .csproj file of your project...